iam/api/QuickStartTest/QuickStartTest.cs

```diff
@@ -3,6 +3,26 @@
 
 public class QuickStartTest
 {
+    private readonly string _project;
+    private readonly string _member;
+    private readonly string _role;
+    
+    public QuickStartTest()
+    {
+        _project = Environment.GetEnvironmentVariable("GOOGLE_PROJECT_ID");
+        _member = "user:YaraArryn.677500@gmail.com";

```

Comment:
I really don't like this, people will know that this user might have some access to a Google owned project and they might try to use that.
What we've done in similar cases is use a service account instead that is created on test startup and deleted on test teardown. That guarantees that the account email is not as easy to guess, that the account is temporary and that even if someone finds one such account they would need the json credentials to use it.

Reply:
Done.